config/crd/bases/hnc.x-k8s.io_hncconfigurations.yaml-e

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
Why there's a `-e` at the end of the file?

Reply:
Yeah, new computer and didn't have kustomize installed, so the makefile used the 'edit' binary (instead of kustomize edit) that created those funny enough. Removed and added to gitignore.